spek-runners/junit5/build.gradle

```diff
@@ -30,3 +21,13 @@ task javadocJar(type: Jar, dependsOn: dokka) {
     from dokka.outputDirectory
     classifier = 'javadoc'
 }
+
+dependencies {
+    compile project(':spek-runtimes:jvm')
+    compileOnly 'org.junit.platform:junit-platform-engine'
```

Comment:
@raniejade this one worries me, it should be `api` or `implementation`, otherwise it'll crash with `ClassNotFoundError` if user won't manually add it (see below)

Or we expect users to always manually add `junit-platform-engine` as test dependency to their projects?

Reply:
We expect users to add them manually, didn't want to be bound to a specific version of JUnit 5 hence `compileOnly`.

Reply:
oook